.github/workflows/scripts/create-transport-tag-manual.sh

```
@@ -0,0 +1,19 @@
+TAG_NAME="transports/v${VERSION}"
+
+echo "🏷️ Creating transport tag: $TAG_NAME"
+
+# Create tag
+git tag "$TAG_NAME" -m "Release transports v${VERSION}"
+
+# Push tag
+git push origin "$TAG_NAME"
+
+echo "✅ Transport tag created and pushed: $TAG_NAME"
+echo "tag-created=true" >> $GITHUB_OUTPUT
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)